charts/jaeger/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 1.20.0
 description: A Jaeger Helm chart for Kubernetes
 name: jaeger
-version: 0.39.7
+version: 0.40.0
```

Comment:
FYI, Jaeger chart has already been updated to 1.21.0 version of jaeger
https://github.com/jaegertracing/helm-charts/blob/master/charts/jaeger/Chart.yaml

Reply:
You can just update the operator if you want on this PR.

Reply:
Sounds good!I just rebased and changed the PR title.